_inc/client/traffic/site-stats.jsx

```diff
@@ -85,89 +86,95 @@ export const SiteStats = moduleSettingsForm(
 				unavailableInDevMode = this.props.isUnavailableInDevMode( 'stats' ),
 				siteRoles = this.props.getSiteRoles();
 			return (
-				<FoldableCard
-					header={ __( 'Site stats' ) }
-					clickableHeader={ true }
-					className={ classNames( 'jp-foldable-settings-standalone', { 'jp-foldable-settings-disable': unavailableInDevMode } ) }
+				<SettingsCard
```

Comment:
Why are you adding SettingsCard back? This was dismissed and replaced with only FoldableCard, which I see it's there, so I think there's no need to add SettingsCard back.